code/game/machinery/cryopod.dm

```diff
@@ -433,28 +434,24 @@
 /obj/machinery/cryopod/MouseDrop_T(atom/movable/O as mob|obj, mob/living/user as mob)
 	if(!istype(user))
 		return
-
 	if(!check_occupant_allowed(O))
 		return
-
 	if(occupant)
 		user << "<span class='notice'>\The [src] is in use.</span>"
 		return
-
 	var/mob/living/L = O
 
 	if(L.stat == DEAD)
 		user << "<span class='notice'>Dead people can not be put into stasis.</span>"
 		return
-
 	for(var/mob/living/carbon/slime/M in range(1,L))
 		if(M.Victim == L)
 			usr << "[L.name] will not fit into the cryo pod because they have a slime latched onto their head."
 			return
 
 	var/willing = null //We don't want to allow people to be forced into despawning.
 
-	if(L.client)
+	if(L.client && !(L.client.is_afk(10 MINUTES)))
```

Comment:
This is a pointless change. Cryo shouldn't process someone if they're still logged in, as such, being able to put cliented folks inside will still see them get stuck.

Adminhelp to victory, let them kick the person.